docker-run-engagement-db-to-analysis.sh

```diff
@@ -0,0 +1,74 @@
+#!/bin/bash
+
+set -e
+
+IMAGE_NAME="$(<configurations/docker_image_name.txt)"
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --incremental-cache-volume)
+            INCREMENTAL_ARG="--incremental-cache-path /cache"
+            INCREMENTAL_CACHE_VOLUME_NAME="$2"
+            shift 2;;
+        --)
+            shift
+            break;;
+        *)
+            break;;
+    esac
+done
+
+# Check that the correct number of arguments were provided.
+if [[ $# -ne 5 ]]; then
+    echo "Usage: $0 
+    [--incremental-cache-volume <incremental-cache-volume>] 
+    <user> <google-cloud-credentials-file-path> <configuration-file> <code-schemes-dir> <data-dir>"
+    exit
+fi
+
+# Assign the program arguments to bash variables.
+USER=$1
+GOOGLE_CLOUD_CREDENTIALS_PATH=$2
+CONFIGURATION_FILE=$3
+CODE_SCHEMES_DIR=$4
+DATA_DIR=$5
+
+CMD="pipenv run python -u engagement_db_to_analysis.py ${INCREMENTAL_ARG} ${USER} \
+    /credentials/google-cloud-credentials.json configuration /data/membership-groups /data/analysis-outputs"
```

Comment:
```suggestion
    /credentials/google-cloud-credentials.json configuration_file /data/membership-groups /data/analysis-outputs"
```

Reply:
Thanks, I think configuration is still the right thing to have here though as this refers to a python module here (the configuration file gets copied in, then used as a module).